reference/conanfile/tools/apple.rst

```diff
@@ -457,8 +459,14 @@ Parameters:
   When ``False`` it will use the settings from the active context. Read more about the
   ``settings_target`` in the :ref:`cross-building section<cross_building>`.
 
-XCRun is a wrapper for the Apple **xcrun** tool used to get information for building. It
-has these properties:
+Public attributes and properties
+++++++++++++++++++++++++++++++++
+
+Public attributes:
+
+  - **settings**: Set of settings used to get the sdk.
```

Comment:
Maybe, this description deserves a little example to avoid doubts about the possible values.

Reply:
I have extended the explanation a bit, I can't think of an example more than getting the public attribute... I don't know if it's worth adding it... WDYT?

Reply:
good enough! 👍